os2phonebook_service/os2phonebook/helpers.py

```diff
@@ -31,7 +30,6 @@ def config_factory():
 
     required_parameters = [
```

Comment:
Should we load the API user/pass variables through here?

Reply:
See 02a3550